spring-social-zotero/src/main/java/org/springframework/social/zotero/api/impl/GroupsTemplate.java

```diff
@@ -252,4 +253,22 @@ public void serializeAsField(Object pojo, JsonGenerator jgen, SerializerProvider
             }
         }
     }
+
+    @Override
+    public void deleteItem(String groupId, Item item, List<String> ignoreFields, List<String> validCreatorTypes)
+            throws ZoteroConnectionException {
+        String url = String.format("groups/%s/%s/%s", groupId, "items", item.getKey());
+
+        HttpHeaders headers = new HttpHeaders();
+        headers.set("If-Unmodified-Since-Version", item.getData().getVersion() + "");
+        JsonNode dataAsJson = createDataJson(item, ignoreFields, validCreatorTypes, false);
+        HttpEntity<JsonNode> data = new HttpEntity<JsonNode>(dataAsJson, headers);
+
+        try {
+            restTemplate.exchange(buildUri(url, false), HttpMethod.DELETE, data, String.class);
```

Comment:
why do you send the item data? It doesn't look like the api needs that, does it?